### PR TITLE
Presto-cloud and book updates

### PR DESCRIPTION
**Pull request by mcknightd**
Added presto cloud to resources page.
image

Added ebook to getting started page, and fixed open href tag the youtube video.
image

### CODE REVIEW

**Comment by dborkar** [comment]
@wenleix @aweisberg , can you please take a look? thank you.

File: website/static/getting-started.html [hunk 1]
```diff
@@ -74,12 +74,22 @@ <h1>Getting Started tutorials provided by the community</h1>
...
    	</ul>
  </ul>
    	</ul>
     <li><a href="https://medium.com/macoclock/installing-prestodb-on-mac-within-3-mins-84720d9b5b45">How to install on Mac in 3 min </a> </li>
+ 
+    <li>
+        <a href=" https://www.oreilly.com/library/view/learning-and-operating/9781492095125/">Learning and Operating Presto</a> published by <a href="https://www.oreilly.com/">O’Reilly Media</a>
+        <ul style="margin-left: 1em;">
+            <li>Early release of O’Reilly book about Presto with introduction and tutorial of real-time analytics use case with Presto and Apache Pinot</li>
+            <li>Includes a forward by Nezih Yigitbasi, Engineering Manager of Presto team at Facebook and founding Presto Foundation Technical Steering Committee Chairperson</li>
```
Comment (aweisberg) [inline website/static/getting-started.html:87]:
```suggestion
            <li>Includes a foreword by Nezih Yigitbasi, Engineering Manager of Presto team at Facebook and founding Presto Foundation Technical Steering Committee Chairperson</li>
```

File: website/static/resources.html [hunk 2]
```diff
@@ -435,6 +435,24 @@ <h2 id="cloud">Presto Cloud</h2>
...
+                <dd><a href="https://ahana.io/" target="_blank">Ahana</a></dd>
+                <dd>
+                    <p>
+                        Ahana Cloud for Presto is the first cloud-native managed service for Presto. This offering is designed to simplify the deployment, management and integration of Presto, with data catalogs, databases and data lakes on Amazon Web Services (AWS). <a href="https://ahana.io/ahana-cloud/" target="_blank">Ahana Cloud for Presto</a> is easy-to-use with no installation, no AMIs or CFTs, and no configuration needed. Via the Ahana console, Presto is deployed within the user’s AWS account, giving users complete control and visibility of clusters and their data. As a managed service, there is no infrastructure to manage, yet you can pay only for Ahana and the EC2 instances that you use. 
+                    <p/>
+                    <p>
+                        Ahana Cloud simplifies Presto. It comes out-of-the-box integrated with built-in catalogs, data sources, and dashboarding tools. Under the hood, Ahana leverages the latest version of Presto and run with Amazon EKS for high scalability, availability, and manageability.
```
Comment (aweisberg) [inline website/static/resources.html:450]:
```suggestion
                        Ahana Cloud simplifies Presto. It comes out-of-the-box integrated with built-in catalogs, data sources, and dashboarding tools. Under the hood, Ahana leverages the latest version of Presto and runs with Amazon EKS for high scalability, availability, and manageability.
```

**Review by aweisberg — APPROVED** [review]
Suggested fixes for two typos. So excited to have the book. I took a look and O'Reilly did require me to register which is unfortunate.

File: website/static/getting-started.html (hunk 1 above)
Comment (mcknightd) [reply website/static/getting-started.html]: Updated in latest commit

File: website/static/resources.html (hunk 2 above)
Comment (mcknightd) [reply website/static/resources.html]: Updated in latest commit

**Comment by wenleix** [comment]
@mcknightd : Can you squash the commits? Thanks!

**Comment by wenleix** [comment]
@mcknightd : We don't do merge and always use rebase. 

I manually pushed the commit this time. In the future, please always use `git pull --rebase` :)

**Comment by wenleix** [comment]
Merged #111, thanks for the contribution!

**Pull request closed without merging**